package.json

```diff
@@ -9,7 +9,7 @@
     "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
     "browser-test-remote-chrome": "nightwatch --config nightwatch.js --env chrome",
     "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "build": "mkdir -p build; browserify src/index.js -o build/app.js",
+    "build": "mkdir -p build; rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js; browserify src/index.js -o build/app.js",
```

Comment:
Is that a good thing to do in the build step? This way it always accesses the internet.